### DISCUSSION TITLE
How to expose a python logger to compute logs?

### DISCUSSION THREAD

**Discussion by adam-bloom** [dagster-io/dagster#23690] [Q&A]
Hi!

I've read a number of related questions, and feel like I'm missing something basic. I have used managed python loggers for before for a python logger that I want to behave as a `context.log` and end up in dagster's event logs. I'm adding another third-party logger that has logs useful for debug/diagnostics. I don't want to chew up dagster event log storage - I have no concern with treating these logs as compute logs that go to stdout/stderr. However, it seems that python loggers not configured as a `managed_python_loggers` just disappear into the ether? What am I missing for adding a basic stdout/stderr handler to all python loggers, and then only redirecting `managed_python_loggers` to the event log? Can I use `dagster_handler_config` for that? The docs (https://docs.dagster.io/concepts/logging/python-logging#configuring-python-log-handlers-) suggested that it only applied to the dagster created loggers, but maybe I missed something.

Thanks!

**Comment by dwink** [comment]
I am struggling with exactly this issue. Did you ever find a solution?

**Reply by adam-bloom** [reply]
Nope. I haven’t solved this - just worked around it. Would love an actual solution!
